tests/playwright/sanity/modules/column/column.test.ts

```diff
@@ -0,0 +1,53 @@
+import { expect } from '@playwright/test';
+import { parallelTest as test } from '../../../parallelTest';
+import WpAdminPage from '../../../pages/wp-admin-page';
+
+test.describe( 'Column tests @column', () => {
+	test.beforeAll( async ( { browser, apiRequests }, testInfo ) => {
+		const context = await browser.newContext();
+		const page = await context.newPage();
+		const wpAdmin = new WpAdminPage( page, testInfo, apiRequests );
+		await wpAdmin.setExperiments( {
+			container: false,
+			'nested-elements': true,
```

Comment:
@nami-p nested is dependent on containers, if you deactivate containers then the nested will be deactivated too. and when you activate nested, then containers are activated.